modules/users/tests/server/user.server.model.tests.js

```diff
@@ -153,6 +150,8 @@ describe('User Model Unit Tests:', function () {
   });
 
   after(function (done) {
-    User.remove().exec(done);
+    User.remove().exec(function() {
```

Comment:
Yeah, if we change the convention to just `.exec(done)` then I'll change it back as it was to be consistent.